src/style.css

```diff
@@ -0,0 +1,75 @@
+body {
+  margin: 0;
+  font-family: Arial, sans-serif;
+}
+
+.box {
+  font-size: 100px;
+  color: white;
+  height: 300px;
+  width: 300px;
+  flex-grow: 1;
+  line-height: 300px;
+  text-align: center;
+}
+
+.box1 {
+  background-color: rgb(255, 0, 0);
+  order: 1;
+}
+
+.box2 {
+  background-color: rgb(204, 0, 0);
+  order: 2;
+}
+
+.box3 {
+  background-color: rgb(153, 0, 0);
+  order: 3;
+}
+
+.box4 {
+  background-color: rgb(102, 0, 0);
+  order: 4;
+}
+
+.box5 {
+  background-color: rgb(51, 0, 0);
+  order: 5;
+}
+
+.box6 {
+  background-color: rgb(0, 0, 0);
+  order: 6;
+}
```

Comment:
no need to repeat orders if they don't change, 
in this case, they already in right order.